autoapi/inheritance_diagrams.py

```diff
@@ -42,11 +42,11 @@ def _import_class(name, currmodule):
         return [target]
 
     if isinstance(target, astroid.Module):
-        classes = []
-        for child in target.get_children():
-            if isinstance(child, astroid.ClassDef):
-                classes.append(child)
-        return classes
+        return [
```

Comment:
I'm not a fan of multi-line list comprehensions. I think we sacrifice readability for a minimal performance benefit. Please could this stay as-is.